app/src/test/java/com/x8bit/bitwarden/data/auth/repository/AuthRepositoryTest.kt

```diff
@@ -5869,23 +5846,19 @@ class AuthRepositoryTest {
         }
 
     @Test
-    fun `validatePin returns ValidatePinResult Error when initialize crypto fails`() = runTest {
+    fun `validatePin returns ValidatePinResult Error when SDK validatePin fails`() = runTest {
```

Comment:
👍  (I scrolled right by this, if you get a notification re: a deleted comment this is why 😳 )